others/valid_no_of_subarrays.cpp

```diff
@@ -0,0 +1,120 @@
+/**
+ * @file
+ * @brief - Finds Total Number of Subarrays whose net product is less than a given Number = k 
+ * [Source](https://leetcode.com/problems/subarray-product-less-than-k/)
+ * 
+ * @description :
```

Comment:
In Doxygen, we use `@details`.
```suggestion
 * @details
```

Reply:
Resolved